fpfilter.pl

```diff
@@ -183,7 +184,8 @@
     if(length($ref) > 1 || length($var) > 1) {
```

Comment:
I don't think these cover all the cases.  What if you have a complex event that's CC ->  AAA? In this, code that would be classified as a deletion instead of skipped, right?